calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx

```diff
@@ -0,0 +1,58 @@
+For example, a user with the Security global role has a broader set of permissions than a user with the Viewer global role.
+These permissions apply to all clusters.
+
+But in some cases these global roles can be too broad.
+
+By creating and assigning custom roles, administrators can be much more discriminating about what permissions they give users.
+For example, you could create a role that gives a user the ability to modify network policy for a particular tier and namespace and gives view access to all other network policies.
+Permissions are assigned on a cluster-by-cluster basis.
+
+## Required permissions for common Calico Cloud features
+
+Certain permissions are required for a user to access common Calico Cloud features.
+
+| Feature area | Required permissions | Notes |
+| --| -- | -- |
+| Alerts | **View** or **Modify Alerts**<br/>and<br/>**View Event Logs** or **View All Logs** | |
+| Anomaly detection | **View** or **Modify Alerts** and **View Event Logs** or **View All Logs**| |
+| Compliance reports | **View Compliance Reports** | |
+| Dashboard | **View All Logs**<br/>and<br/>**View Global Network Sets** or **View Network Sets**<br/>and<br/>(optional) **View Compliance Reports** | These permissions are required for the dashboard to fully populate. Limited metrics are enabled just by having access to a cluster.<br/> |
+| Network policies | **View** or **Modify Policies**<br/>or<br/>**View** or **Modify Global Policies**<br/>and<br/>(optional) **View Audit Logs** or **View All Logs** | The **Policies** permissions are scoped to one or more namespaces. The **Global Policies** permissions apply to the whole cluster. These permissions are also scoped by [policy tier](../network-policy/index.mdx). |
+| Service graph | **View All Logs**<br/>and<br/>**View** or **Modify Network Sets**<br/>and<br/>(optional) **View** or **Modify Packet Captures** | Can be restricted to a namespace or set to all namespaces to see all flows. |
+| Threat feeds | **View** or **Modify Threat Feeds** | |
+| Timeline | **View Event Logs** or **View All Logs** | |
+
+## Before you begin
+
+* You must be signed in to Calico Cloud UI as an administrator.
+
+## Create a custom role and attach permissions
+
+1. Click the user icon <IconUser width="20"/> > **Manage Team**.
+1. Under the **Roles** tab, click **Add Role**, enter a name and description for the custom role, and then click **Save**.
+1. Select the cluster you want to want the role to apply to by clicking **Cluster:** and choosing the cluster.
+1. Locate your new role in the list, select **Action** > **Manage permissions** > **Edit**, and the click **Add Permission**.
+1. Under **Permission**, choose a permission type from the list.
+   Depending on the permission, you may also need to choose a namespace or policy tier.
+1. (optional) Click **Add permission** to add more permissions to your role for this cluster.
+1. Click **Save** to attach these permissions to the role for this cluster.
```

Comment:
No need to introduce "attach"...

1. Click **Save** to save these permissions to the role for this cluster. 
1. (optional) If you want to add the permissions to another cluster, repeat steps 3 to 7 for the cluster.

Reply:
Yes! Thanks!